runbotics/runbotics-orchestrator-ui/src/src-app/views/process/DeleteProcess.tsx

```diff
@@ -74,9 +81,10 @@ const DeleteProcessDialog: VFC<DeleteProcessDialogProps> = (props) => {
 
 type DeleteProcessProps = {
     process: IProcess;
+    disabled?: boolean;
 };
 
-const DeleteProcess = (props: DeleteProcessProps) => {
+const DeleteProcess = ({ process, disabled = false }: DeleteProcessProps) => {
```

Comment:
I know its not yours but it does not sound like function component. Stick to `The Boy Scout Rule` if it does not go far beyond your task